app/src/main/java/nerd/tuxmobil/fahrplan/congress/base/BaseActivity.kt

```diff
@@ -0,0 +1,68 @@
+package nerd.tuxmobil.fahrplan.congress.base
+
+import android.R
```

Comment:
There is a warning about this import in Android Studio. Seems like it wants use to `android.R.id.home`.

Reply:
Interresting, for me it worked using the standard `nerd.tuxmobil.fahrplan.congress.R` 🤔

Reply:
I am not sure if this does the same. We could just leave it as is - then I don't have to spent time on checking it ^^